platforms/i2c/jhd1313m1_driver.go

```diff
@@ -34,10 +34,18 @@ const (
 	LCD_MOVERIGHT           = 0x04
 	LCD_MOVELEFT            = 0x00
 	LCD_2LINE               = 0x08
+	LCD_CMD                 = 0x80
+	LCD_DATA                = 0x40
```

Comment:
I see. Rather then copying the constant names from the C++ code they should be chosen such that they match the [datasheets](http://www.seeedstudio.com/wiki/images/0/03/JHD1214Y_YG_1.0.pdf). The whole point of Gobot is to enable developers to write code for embedded devices in Go instead of C++.

Now, we have 3 different styles in the code:
* Gobot constants that match the spec
* constants taken from the C++ code
* plain values

But as long as the maintainers are fine with that then I'm too :wink: